netutil.py

```diff
@@ -137,7 +137,11 @@ def interfaceUp(interface):
     if rc != 0:
         return False
     inets = filter(lambda x: x.startswith("    inet "), out.split("\n"))
-    return len(inets) == 1
+    if len(inets) == 1:
+        return True
+
+    inet6s = filter(lambda x: x.startswith("    inet6 "), out.split("\n"))
+    return len(inet6s) > 1  # Not just the fe80:: address
```

Comment:
This condition could be moved into the filter; this code is duplicated - so would it be worth to put it into a function?

Reply:
I was thinking to filter the output such that only `inet6` that are not localhost remain. That would make it more explicit which ones you are looking for rather than assuming `fe80::` is in the list or results.

Reply:
So something like:
```py
inet6s = filter(lambda x: x.startswith("    inet6 ") and not x.startswith("    inet6 fe80:"), out.split("\n"))
```
?

Reply:
Yes. Is there a risk that the formatting of the input changes such that it would be better to use `x.strip().find("inet6")` or similar?

Reply:
I don't think so. More recent versions of `ipaddr` still use this format.